stock_scrap_tier_validation/models/stock_scrap.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 Jarsa
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class StockScrap(models.Model):
+    _name = "stock.scrap"
+    _inherit = ["stock.scrap", "tier.validation"]
+    _state_from = ["draft"]
+    _state_to = ["done"]
+
+    _tier_validation_manual_config = False
+
+    def _get_requested_notification_subtype(self):
+        return "stock_scrap_tier_validation.sale_order_tier_validation_requested"
```

Comment:
Mail Subtypes seems wrong, could you cherry pick this commit?
https://github.com/OCA/stock-logistics-workflow/pull/1894/commits/b3d967728668c46b4883476b4e0b2423a0e6ef9f

Reply:
Waiting until this is resolved https://github.com/OCA/stock-logistics-workflow/pull/1894

Reply:
Please rereview #1894 and pick if you agree with the current version.